pages.nl/common/mscore.md

```diff
@@ -0,0 +1,8 @@
+# mscore
+
+> Dit commando is een alias van `musescore`.
+> Meer informatie: <https://musescore.org/handbook/command-line-options>.
+
+- Bekijk documentatie voor het originele commando:
```

Comment:
Hey @Paradact, I just saw that this translation doesn't match our [translation template](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md#nl). Which one should we use?

Reply:
Thanks for the comment, in the template you've linked the Dutch translation does indeed have the definite article ('de', meaning 'the') as well, but the English translation as well as most others I was able to read a little didn't have it.

As for the preposition, both 'voor' and 'van' convey the same meaning, however in my opinion 'van' is slightly more natural.

Do you want me to update this on my fork and resubmit a Pull Request, or is there an easier way of resolving this?

Reply:
@Paradact yeah please, submit a new PR.